traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -443,6 +443,7 @@ UPDATE server SET
 	tcp_port=:tcp_port,
 	type=:server_type_id,
 	upd_pending=:upd_pending,
+	xmpp_id=:xmpp_id,
```

Comment:
we should remove this line to ensure that it's impossible to update (assuming any future bugs in validation logic)

Reply:
I agree, that was the reason I had removed it in #4881

Reply:
Removed `xmpp_id` from the update query in 68c51e362d.